onnxruntime_extensions/tools/add_pre_post_processing_to_model.py

```diff
@@ -185,6 +186,7 @@ def yolo_detection(model_file: Path, output_file: Path, output_format: str = 'jp
     :param onnx_opset: The opset version of onnx model, default(16).
     :param num_classes: The number of classes, default(80).
     :param input_shape: The shape of input image (height,width), default will be asked from model input.
+    :param draw_bboxes: Whether to draw bounding boxes on the image, default(True).
```

Comment:
Would be good for this to explain how this affects the model output (returning scaled bounding boxes vs. a jpg/png image). Should probably also call out that if this is False the `output_format` value becomes unused.